samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52833.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&radio {
+	status = "okay";
+	/* This is an example number of antennas that may be available
+	 * on antenna matrix board.
+	 */
+	dfe-antenna-num = < 10 >;
+	/* This is an example switch pattern that will be used to set an
+	 * antenna for Tx PDU (period before start of Tx CTE).
+	 */
+	dfe-pdu-antenna = <0x1>;
```

Comment:
Minor: One place you have spaces between the value and the brackets (< >), and later you don't.

Reply:
Spaces removed.